js/actions/tests/loadSourceText.js

```diff
@@ -23,8 +23,9 @@ const mockThreadClient = {
   }
 };
 
-describe("loadSourceText", () => {
+describe("loadSourceText", function() {
```

Comment:
`() => {}` will make the context of the body the context of the outer describe, which messes with mocha's funny nested context property where `this` is shared.